### PR TITLE
update examples for batch messages

### PR DESCRIPTION
**Pull request by achilleas-kal**

### CODE REVIEW

File: examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import asyncio
 import logging
+import grpc
+
+import pyinjective.proto.exchange.injective_derivative_exchange_rpc_pb2 as derivative_exchange_rpc_pb
```
Comment (nhannamsiu) [inline examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py:5]: We want to abstract away everything about proto files from users so this doesn't look good

File: examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py [hunk 2]
```diff
@@ -21,19 +25,19 @@ async def main() -> None:
...
-            subaccount_id=subaccount_id,
-            order_hash="0x098f2c92336bb1ec3591435df1e135052760310bc08fc16e3b9bc409885b863b"
-        ),
-        ProtoMsgComposer.OrderData(
-            market_id=market_id,
-            subaccount_id=subaccount_id,
-            order_hash="0x8d4e780927f91011bf77dea8b625948a14c1ae55d8c5d3f5af3dadbd6bec591d"
+    market_id = "0xd0f46edfba58827fe692aab7c8d46395d1696239fdf6aeddfa668b73ca82ea30"
+    orders = []
+    async with grpc.aio.insecure_channel(network.grpc_exchange_endpoint) as channel:
```
Comment (nhannamsiu) [inline examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py:30]: My example shows that they can compose a batch request for different orders on different markets. They will eventually have to write their own logic to filter which order to cancel before composing this, so looping over the all the orders in a single market like this doesn't demonstrate this capability well.

File: examples/chain_client_examples/11_ExchangeMsgBatchCreateSpotLimitOrders.py [hunk 3]
```diff
@@ -15,33 +15,25 @@ async def main() -> None:
...
-            market_id=market_id,
-            subaccount_id=subaccount_id,
-            fee_recipient=fee_recipient,
-            price=27.92,
-            quantity=0.01,
-            isBuy=False
-        ),
-    ]
+    orders = []
+    for order in range(5):
```
Comment (nhannamsiu) [inline examples/chain_client_examples/11_ExchangeMsgBatchCreateSpotLimitOrders.py:27]: Same with this one. My batch orders can be intepreted as "buy when INJ if price is 7.523 and sell INJ if price is 27.92". While the new example might confuse them why they have to add 5 similar orders to pay extra gas when we can do 1 order with quantity = 0.05.

File: examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py (hunk 1 above)
Comment (achilleas-kal) [reply examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py:5]: But realistically nobody is going to take order hashes one by one to append them in a list, they'll have to make this query to cancel all orders from a market or multiple markets given they have 5-10+ orders.

File: examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py (hunk 2 above)
Comment (achilleas-kal) [reply examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py:30]: Same as above

File: examples/chain_client_examples/11_ExchangeMsgBatchCreateSpotLimitOrders.py (hunk 3 above)
Comment (achilleas-kal) [reply examples/chain_client_examples/11_ExchangeMsgBatchCreateSpotLimitOrders.py:27]: Yeah that was just an example, they could loop to make 5 orders for price x=x+1. I don't think they'd manually add this data for every single order given that they place 20 orders+ per market

File: examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py (hunk 1 above)
Comment (nhannamsiu) [reply examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py:5]: Yes, that is covered in another example already. Each of the example demonstrate a discrete action assuming all the inputs are available. The user can compose their script base off these examples, like assembling lego pieces.

For example, deposit funds into subaccount -> fetch all the market ids -> pick the markets they want to trade -> stream the prices -> decide when to create/cancel/batch create/batch cancel/liquidate orders -> get order hashes from markets they just act upon -> continue to stream price and send msgs.

**Pull request closed without merging**